From: David Edmundson <kde@davidedmundson.co.uk>
Date: Mon, 27 Jul 2015 10:54:17 +0000
Subject: Don't refresh the entire window when we render the plotter
X-Git-Url: http://quickgit.kde.org/?p=kdeclarative.git&a=commitdiff&h=7a6a2cda780784e504960c96b4b41241ddf10531
---
Don't refresh the entire window when we render the plotter

This means the window is only updated when the plotter changes, rather
than effectively every possible frame.

Reviewed by: Marco Martin

CCBUG: 348385
---


--- a/src/qmlcontrols/kquickcontrolsaddons/plotter.cpp
+++ b/src/qmlcontrols/kquickcontrolsaddons/plotter.cpp
@@ -709,9 +709,6 @@
 
     // Delete the VBO
     glDeleteBuffers(1, &vbo);
-    if (window()) {
-        window()->update();
-    }
 }
 
 QSGNode *Plotter::updatePaintNode(QSGNode *oldNode, UpdatePaintNodeData *updatePaintNodeData)

